packages/plausible-analytics-bundle/src/plausible-interface/index.ts

```diff
@@ -0,0 +1,41 @@
+import { defineInterface } from '@directus/extensions-sdk';
+import InterfaceComponent from './interface.vue';
+
+export default defineInterface({
+	id: 'plausible-interface',
+	name: 'Plausible Analytics',
+	icon: 'analytics',
+	description: 'Show Plausible Analytics dashboard inside your collection.',
+	component: InterfaceComponent,
+	options: (context) => [
+		{
+			field: 'shareUrl',
+			name: 'Shared Link URL',
+			type: 'string',
+			meta: {
+				width: 'full',
+				interface: 'input',
+				note: 'Create a shared link for your Plausible dashboard and paste the URL here.<br />`(https://plausible.io/share/yourdomain.net?auth=fadfhlkadjl83c)`<br />This URL can be found in the Plausible settings for the domain.<br /> Learn more in the [Plausible documentation](https://plausible.io/docs/shared-links).',
```

Comment:
We should move the sample URL to the input placeholder. Same for all other relevant occurrences